### ISSUE TITLE
equalTo and error message

### ISSUE THREAD

**Issue by watery** [jzaefferer/jquery-validation#1607]
When the `equalTo` validation rule fails, the error message is fixed, it doesn't report to which field the value has to be equal to.

**Comment by staabm** [comment]
Since we only know a selector to the equalTo element, I dont know how we should generate a user readable/friendly message.

Any suggestion what the expecred behavior should be?

**Comment by Arkni** [comment]
@watery 
Why not to use your own message for that, something like that:
```js
$( "#myform" ).validate({
  rules: {
    password: "required",
    password_again: {
      equalTo: "#password"
    }
  },
  messages: {
    password_again: {
      equalTo: "Please enter the same value as in `password` element"
    }
  }
});
```

**Comment by staabm** [comment]
@Arkni thats what Inwanted to suggest between the lines :-), thanks.

@watery Will close this issue now, feel free to discuss it further.
